addons/website_slides/models/slide_channel.py

```diff
@@ -1052,7 +1052,7 @@ def _send_share_email(self, emails):
                 base_url=record.get_base_url(),
             )
             email_values = {'email_to': emails}
-            if self.env.user.has_group('base.group_portal'):
+            if self.env.user._is_portal():
```

Comment:
Do you make this change for consistency ? This change is not necessary because the env.user is always sudoed, right ? (same for `_is_public`)

Reply:
That's indeed for consistency.